napari/_app_model/_tests/test_app.py

```diff
@@ -1,19 +1,24 @@
-from napari._app_model import get_app
+import pytest
+
+from napari._app_model import get_app, get_app_model
 from napari.layers import Points
 
 
 def test_app(mock_app):
     """just make sure our app model is registering menus and commands"""
-    app = get_app()
+    app = get_app_model()
     assert app.name == 'test_app'
     assert list(app.menus)
     assert list(app.commands)
     # assert list(app.keybindings)  # don't have any yet
+    with pytest.warns(FutureWarning):
```

Comment:
Please use `match` argument with matching string to check if a proper future warning is emmited